app/views/service_providers/_form.html.slim

```diff
@@ -1,8 +1,21 @@
+- edit_form ||= false
```

Comment:
but if we don't need the if / else, per my other comment, then no need to use `content_for` !